web/models/users.ex

```diff
@@ -135,7 +137,7 @@ defmodule HexWeb.Users do
     case Repo.transaction(multi) do
       {:ok, %{email: email}} ->
         user = with_emails(%{user | emails: %Ecto.Association.NotLoaded{}})
-        Mailer.send_verification_email(user, email)
+        Mail.verification(user, email) |> Mailer.deliver_now_throttled(user, email)
```

Comment:
`Mailer.deliver_now_throttled(user, email)` -> `Mailer.deliver_now_throttled`

I believe this is the last required change - thank you for your help!

PS it's easier to review when you don't force push commits but add new ones - we'll squash it anyway when merging